digital-deck-server/model/player.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// Create the Player Schema
+const PlayerSchema = new Schema({     // mongodb will create an auto generated id, so we don't need to create one here
+    name: String,
+    cards: [ {type:Schema.Types.ObjectId, ref: 'Card'}] , 
```

Comment:
Related to my comment in `card.js`, this can simply be a string of card ids that the player has, eg. `1,10,3,50`